### DISCUSSION TITLE
hackintosh/updates/2022/02/10/acidanthera-february

### DISCUSSION THREAD

**Discussion by giscus** [dortania/dortania.github.io#32] [Comments]
# hackintosh/updates/2022/02/10/acidanthera-february

A message from vit9696 Changelogs (coming soon!) Note: This post was originally supposed to go up on 2022-02-07. We are sorry for the delay. A message from vit9696 Good evening, We have just released one more stability upgrade: Thanks to @mikebeaton, OpenCore will finally work correctly with SIP values, OpenLinuxBoot will no longer crash on partially valid setups, and AudioDxe will work as normal with older macOS versions. @Goldfish64 updated RestrictEvents to work with older macOS versions and also fixed its configuration abilities, making them more logical. With the help of @Sergey-Galan and others, more codec controller patches were updated to the new format, resolving some HDMI issues in AppleALC. @mikebeaton also updated alc-verb to more closely resemble hda-verb behaviour in Linux to increase the interoperability. AirportBrcmFixup was updated by @lvs1974 to be more stable on macOS 12.1+. @Ph-FrX fixed several backlight issues in WhateverGreen on Ice Lake platforms. While that makes it for today, we are currently working on improving bootloader security and reliability. Stay safe, — Vit Changelogs OpenCore 0.7.8 Lilu 1.6.0 FeatureUnlock 1.0.6 WhateverGreen 1.5.7 gfxutil 1.83b RestrictEvents 1.0.7 AirportBrcmFixup 2.1.4 CpuTscSync 1.0.6 AppleALC 1.6.9

https://dortania.github.io/hackintosh/updates/2022/02/10/acidanthera-february.html

**Comment by ThatStella7922** [comment]
based

**Comment by ccnmxns** [comment]
Voilà ! Now my laptop works fine with Monterey!

PS: Codec ALC294 layout-id 44 for ASUS UX534FAC by @narcyzzo also works in ASUS UX431FAC :-)

**Comment by jcckg** [comment]
# bloody astonishing

**Comment by sunwoo-kim** [comment]
Hi, I didn't know where to put this comment to find you, but I wanted to know if you guys could include Macbook Pro 12,1 (Early 2015) into supported Macbook for Opencore Legacy Patcher. The reason is that this macbook doesn't support universal control, and there is no way to patch it so that it is enabled :( Much thanks.

**Comment by chriswayg** [comment]
Over a period of two years of helpful posts you have created the expectation of regular monthly updates, but then suddenly stopped. There has not been an update in almost half a year. What happened? When will you publish the next *Acidanthera Updates*, or have these Updates been discontinued?
